client/components/Buttons/LikeButton.vue

```diff
@@ -23,7 +23,9 @@ export default Vue.extend({
   },
   data() {
     return {
-      isFavorite: false
+      isFavorite: false,
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
+      unsubscribe(): void {}
```

Comment:
We won't need this as soon as the item store is properly implemented, that's why I intentionally skipped them in my PR. Doesn't harm to have this during the transition period.

Reply:
Ok that's why :P